lib/commands/publish.js

```diff
@@ -130,11 +130,18 @@ class Publish extends BaseCommand {
     }
 
     const resolved = npa.resolve(manifest.name, manifest.version)
+    const registry = npmFetch.pickRegistry(resolved, opts)
+
+    const latestVersion = await this.#latestPublishedVersion(resolved, registry)
+    const latestTagIsGreater = !!latestVersion && semver.gte(latestVersion, manifest.version)
+
+    if (latestTagIsGreater && isDefaultTag) {
+      throw new Error('Cannot publish a lower version without an explicit dist tag.')
```

Comment:
Should we just call this "tag" since that's what the config is `--tag`?

I know in the code we call it dist-tag cause that's what actually is, and `npm dist-tag` is the command, but `--tag` is the flag.

I'm probably overthinking this a bit but I know in the other PR for pre-release guarding we said `--tag` in the error.

Reply:
updating!